source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+- Allow a couple of minutes to get the activation email. All changes in the developer portal take a couple of minutes before the change is implemented.
+
+{% endimportant %}
+
+{% details "I have manually disabled My Home Assistant" %}
+
+If you don't have [My Home Assistant](/integrations/my) on your installation,
+you can use `<HOME_ASSISTANT_URL>/auth/external/callback` as the redirect URI
+instead.
+
+The `<HOME_ASSISTANT_URL>` must be the same as used during the configuration/
+authentication process.
+
+Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://homeassistant.local:8123/auth/external/callback`.
+
+{% enddetails %}
+
+{% include integrations/config_flow.md %}
+
+The integration configuration will ask for the *Client ID* and *Client Secret* created above. See [Application Credentials](/integrations/application_credentials) for more details.
+
+## Supported functionality
+
+{% note %}
+
+- The entities' availability depends on the appliance type and the generation of the product, and the appliance might not support all the entities for its type.
+- Products from professional and semi-professional series are generally not supported due to the limitations in the Miele 3rd party API.
+- Some appliances don't report data while they are turned off, so corresponding entities will not appear in the Miele integration after loading until the appliances are turned on.
+{% endnote %}
+
+### Sensor
+
+{% details "List of sensors" %}
+
+- **Operation state**:
+  - **Status**: Represents the current operation state of the device. The default entity name is just the appliance type e.g., "Dishwasher".
+  - **Temperature**: Represents the current temperature in refrigerators, freezers and ovens. Entities are created for up to 3 zones depending on the device capabilities.
+{% enddetails %}
+
+## Automation examples
+
+Get started with these automation examples
+
+### Send a notification when the appliance ends the program
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
+alias: "Notify when program ends"
+triggers:
+  - trigger: state
+    entity_id:
+      - sensor.washing_machine
+    to: program_ended
+actions:
+  - service: notify.notify
+    data:
+      message: "The appliance has finished the program."
+```
+
+{% endraw %}
+{% enddetails %}
+
+## Data updates
+
+This integration uses server-sent events from the Miele API to receive live updates from the appliances.
+When the configuration entry is loaded or after a streaming error (for example after disconnection), the integration will request all data (such as appliance info, available commands, programs, settings, and status) for all appliances.
+If a new appliance is added to the account, the integration will request data for the new appliance and expose the related entities automatically after a reload of the integration.
```

Comment:
It all sounds promising until the reload part of the integration, I would remove this as you might end up doing this dynamically anyway